WordPress/Classes/ViewRelated/Reader/ReaderMenuViewController.swift

```diff
@@ -75,6 +93,21 @@ import WordPressShared
     // MARK: - Configuration
 
 
+    func setupRefreshControl() {
+        if refreshControl != nil {
```

Comment:
Hmm.  A guard statement would still be a nil check `guard refreshControl == nil else {`, and we'd still have the optional syntax after the assignment because that's what the type calls for.   Not sure what the gain would be. Maybe I'm misunderstanding?